packages/nextjs/src/server/clerkMiddleware.ts

```diff
@@ -178,12 +198,37 @@ export const clerkMiddleware = ((...args: unknown[]): NextMiddleware | NextMiddl
       } catch (e: any) {
         handlerResult = handleControlFlowErrors(e, clerkRequest, request, requestState);
       }
+      if (options.contentSecurityPolicy) {
+        const result = createCSPHeader(
```

Comment:
do we want to do this by default? maybe in a future release?

Reply:
I think in the future major that would make sense. It would definitely be a big breaking change for anyone loading external assets, like analytics scripts, fonts and such

Reply:
Do we keep a list of changes we want to go into the next upcoming major? I wouldn't want to forget about this one